sources/assets/Stride.Core.Packages/Stride.Core.Packages.csproj

```diff
@@ -2,7 +2,7 @@
   <Import Project="..\..\targets\Stride.props" />
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <TargetFramework>$(StrideEditorTargetFramework)</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
You might want undo those changes and instead cherry-pick the commit 76000fc2 from #1908 .